### PR TITLE
Feature/update map ui

### PR DESCRIPTION
**Pull request by GiftLanga**
implemented heat map background gradient color

### CODE REVIEW

File: waterlog/src/Components/Map.js [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
-const northEast = [-25.782110, 28.338325];
-const maxIntensity = 5;
-const centerPosition = [-25.783425, 28.336046];
-const defaultZoom = 16;
+import { generateMapIcons, levelToIntensity, mapOptions } from "./../utils";
 
 function getHeatMapData({ monitorsCoordinates, segmentCoordinates }) {
-  let monitorMapData = monitorsCoordinates.map(mon => [mon.lat, mon.long, levelToIntensity(mon.faultLevel, maxIntensity)]);
-  let segmentMapData = segmentCoordinates.map(seg => [seg.lat, seg.long, levelToIntensity(seg.faultLevel, maxIntensity)]);
+  let monitorMapData = monitorsCoordinates.map(mon => [mon.lat, mon.long, levelToIntensity(mon.faultLevel, mapOptions.maxIntensity)]);
```
Comment (FredJEhlers) [inline waterlog/src/Components/Map.js]: Split over multiple lines

Comment (FredJEhlers) [reply waterlog/src/Components/Map.js]: Line 13 and 14

File: waterlog/src/Components/MapFullScreen.js [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
+import {
+  generateMapIcons,
+  levelToIntensity,
+  mapOptions
+} from "../utils";
 
 function getHeatMapData({ monitorsCoordinates, segmentCoordinates }) {
-  let monitorMapData = monitorsCoordinates.map(mon => [mon.lat, mon.long, levelToIntensity(mon.faultLevel, maxIntensity)]);
-  let segmentMapData = segmentCoordinates.map(seg => [seg.lat, seg.long, levelToIntensity(seg.faultLevel, maxIntensity)]);
+  let monitorMapData = monitorsCoordinates.map(mon => [mon.lat, mon.long, levelToIntensity(mon.faultLevel, mapOptions.maxIntensity)]);
```
Comment (FredJEhlers) [inline waterlog/src/Components/MapFullScreen.js]: Split over multiple lines (Again both)

File: waterlog/src/utils.js [hunk 3]
```diff
@@ -159,3 +160,17 @@ export function levelToIntensity(level, maxIntensity = 5) {
       return 0;
   }
 }
+
+const sW = [-25.784510, 28.334360];
```
Comment (FredJEhlers) [inline waterlog/src/utils.js]: Please use descriptive variable names

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/Components/Map.js (hunk 1 above)
Comment (GiftLanga) [reply waterlog/src/Components/Map.js]: done

File: waterlog/src/Components/MapFullScreen.js (hunk 2 above)
Comment (GiftLanga) [reply waterlog/src/Components/MapFullScreen.js]: done

File: waterlog/src/utils.js (hunk 3 above)
Comment (GiftLanga) [reply waterlog/src/utils.js]: done

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**